src/Orckestra.Composer.MyAccount/Services/MembershipViewService.cs

```diff
@@ -189,7 +189,8 @@ public virtual async Task<LoginViewModel> LoginAsync(LoginParam param)
 
             if (loginResponse)
             {
-                var user = Membership.GetUser(userName, true);
+                var profileSettings = await CustomerSettings.GetProfileSettingsAsync().ConfigureAwait(false);
+                var user = profileSettings.UseEmailAsUsername ? Membership.GetUserByEmail(userName, true) : Membership.GetUser(userName, true);
                 if (user != null && user.ProviderUserKey is Guid && !Guid.Empty.Equals(user.ProviderUserKey) && !user.IsLockedOut)
                 {
```

Comment:
could you please try first get user by username as was before - Membership.GetUser(userName, true);, then if user is null and profileSettings.UseEmailAsUsername then try to get by email Membership.GetUserByEmail(userName, true) 
To avoid have always additional API calls when UseEmailAsUsername  is true,
In most cases when UseEmailAsUsername   = thue, the  Membership.GetUser(userName, true); will return user
The problem just when we switch from UseEmailAsUsername   = false to true.